ush/config.sh.RRFS_CONUS_3km

```diff
@@ -0,0 +1,161 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+#RESERVATION="rrfsdet"
+EXPT_BASEDIR="YourOwnSpace"
+EXPT_SUBDIR="RRFS_CONUS_3km"
+
+if [[ -n $RESERVATION ]] ; then
+  ACCOUNT=rtrr
+  SERVICE_ACCOUNT=nrtrr
+  PARTITION_DEFAULT=kjet
+  PARTITION_FCST=kjet
+  PARTITION_GRAPHICS=kjet,xjet,tjet,ujet,vjet
+  PARTITION_ANALYSIS=kjet
+fi
+
+if [[ $MACHINE == "hera" ]] ; then
+  ACCOUNT="wrfruc"
+  PARTITION_DEFAULT=""
+  PARTITION_FCST=""
+  QUEUE_ANALYSIS="batch"
+fi
+if [[ $MACHINE == "hera" ]] ; then
+  ACCOUNT="wrfruc"
+  PARTITION_DEFAULT=""
+  PARTITION_FCST=""
+  QUEUE_ANALYSIS="batch"
+fi
+if [[ $MACHINE == "orion" ]] ; then
+  ACCOUNT=ztrr
+  QUEUE_ANALYSIS="batch"
+fi
+
+VERBOSE="TRUE"
+
+RUN_ENVIR="nco"
+PREEXISTING_DIR_METHOD="upgrade"
+
+PREDEF_GRID_NAME=RRFS_CONUS_3km
+ADDNL_OUTPUT_GRIDS=( "hrrr" )
+
+TILE_LABELS="CONUS REGIONS"
+TILE_SETS="full NE,NC,NW,SE,SC,SW"
+
+DO_DACYCLE="TRUE"
+#DO_SPINUP="TRUE"
+DO_RETRO="TRUE"
+#LBCS_ICS_ONLY="TRUE"
+DO_NONVAR_CLDANAL="TRUE"
+DO_REFL2TTEN="TRUE"
+RADARREFL_TIMELEVEL=(0 15 30 45)
+FH_DFI_RADAR="0.0,0.25,0.5"
+
+QUILTING="TRUE"
+CCPP_PHYS_SUITE="FV3_HRRR"
+
+EXTRN_MDL_ICS_OFFSET_HRS="3"
+BOUNDARY_LEN_HRS="21"
+LBC_SPEC_INTVL_HRS="1"
+EXTRN_MDL_LBCS_OFFSET_HRS="0"
+EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS="0"
+
+DATE_FIRST_CYCL="20210512"
+DATE_LAST_CYCL="20210513"
+CYCL_HRS=( "00" "12" )
+CYCL_HRS_SPINSTART=("03" "15")
+CYCL_HRS_PRODSTART=("03" "15")
+CYCLEMONTH="5"
+#CYCLEDAY="12-14"
+
+PREEXISTING_DIR_METHOD="upgrade" # "rename"
```

Comment:
This is a duplicate. PREEXISTING_DIR_METHOD is already set at line 36.

Reply:
removed one